src/test/msgr/test_msgr.cc

```diff
@@ -37,6 +37,14 @@
 
 typedef boost::mt11213b gen_type;
 
+#include "common/dout.h"
```

Comment:
because I want to get the right sequence which mixes ceph logginer and cerr. Otherwise, cerr output make the logs a little disordered.